.gitmodules

```diff
@@ -35,3 +35,7 @@
 	path = condition-plugins
 	url = ../../data-integrations/condition-plugins.git
 	branch = develop
+[submodule "azure"]
+	path = azure
+	url = https://github.com/data-integrations/azure
```

Comment:
Maybe change this to `../../data-integrations/azure` to match the format for the other URLs?

Reply:
Fixed